### ISSUE TITLE
.npmignore in blueprint replaces projects .npmignore

### ISSUE THREAD

**Issue by rwjblue** [ember-cli/ember-cli-blueprint-test-helpers#3]
```
% ember install ember-cli-blueprint-test-helpers
version: 1.13.12
Installed packages for tooling via npm.
installing ember-cli-blueprint-test-helpers
[?] Overwrite .npmignore? Diff
Index: /Users/rjackson/Source/javascript/ember-data/.npmignore
===================================================================
--- /Users/rjackson/Source/javascript/ember-data/.npmignore
+++ /Users/rjackson/Source/javascript/ember-data/.npmignore
@@ -1,15 +1,1 @@
-bower_components/
-tests/
-tmp/
-dist/
-
-.bowerrc
-.editorconfig
-.ember-cli
-.travis.yml
-.npmignore
-**/.gitkeep
-bower.json
-Brocfile.js
-testem.json
-*.gem
+blueprint-nodetest.js
\ No newline at end of file

[?] Overwrite .npmignore? (Yndeh)
```

**Comment by trabus** [comment]
@rwjblue Should I remove `.npmignore` from the files folder and add to the existing file (or create first if it doesn't exist) then add the line to `.npmignore` manually?

**Comment by trabus** [comment]
closing, this was resolved by #15
